QuestionBot/QuestionBot/Model/QuestionMessageListener.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuestionBot.Model{
+
+    public class QuestionMessageListener : IMessageListener{
+
+        private IStore _questionDataStore;
+
+        public QuestionMessageListener(IStore store){
+            _questionDataStore = store;
+        }
+
+        public void ReceiveMessage(string input){
+            const string questionCommand = "/question";
+            const string answerTag = "/answer";
+
+            if (input.StartsWith(questionCommand)){
+                string questionText = input.Remove(0, questionCommand.Length);
+                questionText = questionText.TrimStart(' ');
+
+                if (!String.IsNullOrWhiteSpace(questionText)){
```

Comment:
I suppose not, if someone were to enter just '/question' then it would remove /question and leave just an empty string.

From String.Trim MSDN (assuming TrimStart would act similarly in this case):

> If the current string equals Empty or all the characters in the current instance consist of white-space characters, the method returns Empty.

I could instead do `questionText.Equals(String.Empty)`